startup.krun

```diff
@@ -60,7 +60,10 @@ def patch_platform(platform):
         if "platform_sanity_check" in args[-3]:
             return prepend_args + args
 	return prepend_args + ["startup_runners/outer_startup_runner_c"] + args
-    assert Cls.__dict__.get('bench_cmdline_adjust') is None
+    old_bench_cmdline_adjust = Cls.__dict__.get('bench_cmdline_adjust')
```

Comment:
So why is bench_cmdline_adjust non-None the second time around? The patching somehow persists between runs?